### PR TITLE
feat: fantom support

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did
partial: #4 

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by fubuloubu** [comment]
some issue with how ape-fantom is getting registered

**Review by unparalleled-js — APPROVED** [review]
lgtm

File: ape_etherscan/explorer.py [hunk 1]
```diff
@@ -10,14 +10,16 @@
...
     def get_transaction_url(self, transaction_hash: str) -> str:
-        return f"{get_etherscan_uri(self.network.name)}/tx/{transaction_hash}"
+        etherscan_uri = get_etherscan_uri(self.network.ecosystem.name, self.network.name)
+        return f"{etherscan_uri}/tx/{transaction_hash}"
 
     @property
-    def _client_factory(self) -> ClientFactory:
-        return ClientFactory(self.network.name)
+    def _client_factory(self):
+        return ClientFactory(self.network.ecosystem.name, self.network.name)
```
Comment (NotPeopling2day) [inline ape_etherscan/explorer.py:22]: Does this need a return type?

**Review by NotPeopling2day — COMMENTED** [review]
LGTM

**Review by NotPeopling2day — APPROVED** [review]

**Pull request merged**